src/providers/dataProvider/index.ts

```diff
@@ -0,0 +1,45 @@
+import localStorageProvider from 'ra-data-local-storage';
+import { withLifecycleCallbacks } from 'react-admin';
+import constants from '../../constants';
+import { AuditType, trackEvent } from '../../utils';
+import users from './users';
+
+export const dataProvider = localStorageProvider({
```

Comment:
@salony75way - I think it's more complex to configure initially, but I would like us to try `ra-data-localforage`, since it seems a more modern, versatile way of using local-storage.  
https://github.com/marmelab/react-admin/tree/master/packages/ra-data-localforage

Reply:
Done.